internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
```

Comment:
Ensure that the 'env' package properly handles errors and edge cases when parsing environment variables. Consider checking for other possible parsing errors that may not be covered by the current error output.